align/schema/hacks.py

```diff
@@ -98,31 +98,30 @@ def translate(self, solver):
         # Load constraints
         yield from self.constraints.translate(solver)
 
-    def verify(self, formulae=None):
-        if formulae is None:
+    def verify(self, constraint=None):
```

Comment:
Needed to change the API to fix the error. Constraint translation is now done inside the 'verify' routine so that the last added constraints are available to the exception handler.